2015-09-31-habitica-habit-building-program.md

```diff
@@ -0,0 +1,204 @@
+
+date:   2015-07-31 22:00:00
+categories: Productivity
+cover_image: habitica/intro.png
+cover_image_caption: "Motivate yourself to do anything."
+news_keywords: "Productivity, Application, Game, Todo list, Community"
+post_images: [
+    "habitica/logo.png",
+    "habitica/health_xp_mana.png",
+    "habitica/classes.png",
+    "habitica/items.png",
+    "habitica/party.png",
+    "habitica/badges.png",
+    "habitica/pm.png",
+]
+
+description: "A brief look into Habitica, one of the best gamified productivity
+applications, and a few tips"
+
+---
+
+`Well... I'm already using Habitica, so...`<br />
+Wait! I've got a gift for you :smile: Go directly to [the tips](#tips).
+
+# The minimalist guide to Habitica
+
+[Habitica](https://habitica.com) is a todo-list
+application inspired from the [Get Things Done](http://gettingthingsdone.com/)
+productivity methodology.
+
+You basically have 4 columns:
+
+* **Habits**: for activities that could be repeated several times a day.
+* **Dailies**: for recurring activities.
+* **Todo**: for one-time tasks.
+* **Rewards**: for leisure activities available after working hard on your tasks.
+
+It's  simple to use, accepts the [Markdown](http://daringfireball.net/projects/markdown/) 
+markup language and [Emoji](http://emoji-cheat-sheet.com/) emoticons.
+
+
+Although its todo-list features have increased a lot over the time, I believe the
+strenght of Habitica comes from the game that wraps the tool, and the community
+that gives it life.
+
+As a fact, I've tried many todo-list tools before Habitica, and always gave
+up on them (missing features, not intuitive enough, or simply because I couldn't
+build the habit of checking them regularly and forgot them). But when I started
+using Habitica back in its beta days, although it was cribbled with bugs, I
+sticked to it because I was having fun and had met very friendly people there.
+
+{: .clear}
+![Habitica Logo](/images/habitica/logo.png "New logo looking like a Gryphon"){: .float_left}
+Formely named **HabitRPG**, the application was renamed **Habitica** on July 31th 2015.
+The old name was difficult to pronunce and confused by people
+(HabitRG? HabitGRG?), so they decided to name the application after the land of
+Habitica, where all these adventures take place.
+
+The [Gamification](https://en.wikipedia.org/wiki/Gamification) comes in
+multiples aspects:
+
+* **Health**, **Experience**, **Mana**: Like any [RPG
+  game](https://en.wikipedia.org/wiki/Role-playing_game), your avatar will
+  evolve as you complete tasks, and will die if you're slacking off.
+
+  {: .centered}
+  ![Health, XP and Mana bars](/images/habitica/health_xp_mana.png)
+* **Attribute points and Class System**: Do you want to complete quests faster,
+  get more gold, avoid dying? Smartly distribute your points in [Strenght,
+  Intelligence, Constitution and Perception](http://habitica.wikia.com/wiki/Character_Attributes),
+  and select the most appropriate class for your avatar
```

Comment:
Not sure if I should develop this part. But I'll add a link to the tips section, and add more advanced tips on what class to use in which circumstance, thanks!.